steward/src/commands.rs

```diff
@@ -46,9 +44,6 @@ pub enum StewardCmd {
     #[clap(subcommand)]
     Orchestrator(orchestrator::OrchestratorCmd),
     SignDelegateKeys(sign_delegate_keys::SignDelegateKeysCmd),
-    #[clap(subcommand)]
-    Tx(tx::TxCmd),
-    AllowErc20(allow_erc20::AllowERC20),
```

Comment:
If we're deleting these, we should delete the commands themselves and update the main README to not reference them.